torch_np/_wrapper.py

```diff
@@ -114,14 +121,17 @@ def arange(start, stop=None, step=1, dtype=None, *, like=None):
         # arange(stop)
         stop = start
         start = 0
-    return asarray(torch.arange(start, stop, step, dtype=dtype))
+    torch_dtype = _dtypes.torch_dtype_from(dtype)
+    return asarray(torch.arange(start, stop, step, dtype=torch_dtype))
 
 
 def empty(shape, dtype=float, order='C', *, like=None):
     _util.subok_not_ok(like)
     if order != 'C':
         raise NotImplementedError
-    return asarray(torch.empty(shape, dtype=dtype))
+    dtype = _dtypes.dtype(dtype)
+    torch_dtype = _dtypes.torch_dtype_from(dtype)
```

Comment:
We may want to put these two into a function, given that we're going to use this pattern left right and centre.